test/07_upload_files.ts

```diff
@@ -0,0 +1,399 @@
+
+            expect(responseInvalid.status).to.eqls(400);
+        });
+
+        it("Requires mimeType param to be string", async () => {
+            const responseInvalid = await supertest(app)
+                .post("/api/upload/files")
+                .set({ accesstoken: globals.userToken })
+                .send({ ...validParams, mimeType: undefined });
+
+            const responseInvalidNotString = await supertest(app)
+                .post("/api/upload/files")
+                .set({ accesstoken: globals.userToken })
+                .send({ ...validParams, mimeType: 42 });
+
+            const responseValid = await supertest(app)
+                .post("/api/upload/files")
+                .set({ accesstoken: globals.userToken })
+                .send({ ...validParams, mimeType: "string" });
+
+            expect(responseInvalid.status).to.eqls(400);
+            expect(responseInvalidNotString.status).to.eqls(400);
+            expect(responseValid.status).to.eqls(200);
+        });
+
+        it("If defined, requires fileHash param to be string", async () => {
+            const responseNoParam = await supertest(app)
+                .post("/api/upload/files")
+                .set({ accesstoken: globals.userToken })
+                .send({ ...validParams, fileHash: undefined });
+
+            const responseInvalidNotString = await supertest(app)
+                .post("/api/upload/files")
+                .set({ accesstoken: globals.userToken })
+                .send({ ...validParams, fileHash: 42 });
+
+            const responseValid = await supertest(app)
+                .post("/api/upload/files")
+                .set({ accesstoken: globals.userToken })
+                .send({ ...validParams, fileHash: "string" });
+
+            expect(responseInvalidNotString.status).to.eqls(400);
+            expect(responseNoParam.status).to.eqls(200);
+            expect(responseValid.status).to.eqls(200);
+        });
+
+        it("If defined, requires relationId param to be number", async () => {
+            const responseNoParam = await supertest(app)
+                .post("/api/upload/files")
+                .set({ accesstoken: globals.userToken })
+                .send({ ...validParams, relationId: undefined });
+
+            const responseInvalidNotNumber = await supertest(app)
+                .post("/api/upload/files")
+                .set({ accesstoken: globals.userToken })
+                .send({ ...validParams, relationId: "42" });
+
+            const responseValid = await supertest(app)
+                .post("/api/upload/files")
+                .set({ accesstoken: globals.userToken })
+                .send({ ...validParams, relationId: 5 });
+
+            expect(responseInvalidNotNumber.status).to.eqls(400);
+            expect(responseNoParam.status).to.eqls(200);
+            expect(responseValid.status).to.eqls(200);
+        });
+
+        it("Returns error if hash is not matching", async () => {
```

Comment:
@kenyasue tu je taj test

Reply:
Ali bih htio da radiš baš test sa pravi file, jer ovako ne testira se slučaj kad se spaja file. To je dosta bitno jer zna biti problema kod spajanja.